inits.json

```diff
@@ -0,0 +1,54 @@
+{
+    "user_info": [
+        {
+            "Directory with FITS files": "./sample-data/HatP32Dec202017",
+            "Directory to Save Plots": "./sample-data",
+            "Directory of Flats": "n",
+            "Directory of Darks": "n",
+            "Directory of Biases": "n",
+
+            "AAVSO Output?": "y",
```

Comment:
I'm thinking that we should _always_ do the AAVSO output and then if the user does not have an AAVSO Observing Account Number, they just put `none` or something else applicable. That way they can have the output file already on their computer and if they reconsider and want to upload to AAVSO, they don't have to re-run EXOTIC again.

Reply:
^ this can be a new opened ticket for a future release

Reply:
Done in this push. Users must have to put in their AAVSO code or N/A if none.